metadata.py

```
@@ -104,10 +104,15 @@ def find_cover(path: str) -> Optional[None]:
             if not image_files:
                 logging.error("Empty archive.")
                 return
-            image_files.sort(key=sort_imgs)
-            first_image = zip_ref.read(image_files[0])
+            cover_files = [f for f in image_files if os.path.splitext(os.path.basename(f))[0].lower() == "cover"]
```

Pylint (3.3.7)

[convention] Line too long (113/100)

(C0301)